types/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name    = "sim_types"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Comment:
patch versions used, should be changed to major + minor only

Reply:
564ebfb24e324f1c98f47b94587a72cf0d550abf